desktop/mac/permission-requirements.md

```diff
@@ -7,15 +7,85 @@ redirect_from:
 - /desktop/mac/privileged-helper/
 ---
 
-This page contains information about the permission requirements for running and installing Docker Desktop on Mac, the functionality of the privileged helper process `com.docker.vmnetd` and the reasoning behind this approach. 
+This page contains information about the permission requirements for running and installing Docker Desktop on Mac.
 
 It also provides clarity on running containers as `root` as opposed to having `root` access on the host.
 
 ## Permission requirements
 
-In the default set up flow, Docker Desktop for Mac doesn't require root privileges for installation but does require `root` access to be granted on the first run. The first time that Docker Desktop is launched the user receives an admin prompt to grant permissions for a privileged helper service to be installed. For subsequent runs, no `root` privileges are required.
+Docker Desktop for Mac is run as an unprivileged user. However, certain functionalities may be required for Docker Desktop to perform a limited set of privileged configurations such as:
```

Comment:
`However, certain functionalities may be required for Docker Desktop to perform a limited set of privileged configurations` 
doesn't sound right. Perhaps:
`However, certain functionalities may require Docker Desktop to perform a limited set of privileged configurations` ?